### PR TITLE
Prompt the user before deleting table rows

### PR DESCRIPTION
**Pull request by fkm**

### CODE REVIEW

File: translations/de.yaml [hunk 1]
```diff
@@ -8,8 +8,10 @@ caluma:
   form:
     optional: "Optional"
     save: "Speichern"
+    delete: "Löschen"
     selectFile: "Klicken um Datei hochzuladen"
     changeFile: "Klicken um andere Datei hochzuladen"
+    deleteRow: "Wollen Sie diese Zeile löschen?"
```
Comment (czosel) [inline translations/de.yaml]:
```suggestion
    deleteRow: "Möchten Sie diese Zeile wirklich löschen?"
```

File: translations/en.yaml [hunk 2]
```diff
@@ -8,8 +8,10 @@ caluma:
   form:
     optional: "Optional"
     save: "Save"
+    delete: "Delete"
     selectFile: "Click to upload a file"
     changeFile: "Click to upload a different file"
+    deleteRow: "Do you want to delete this row?"
```
Comment (czosel) [inline translations/en.yaml]:
```suggestion
    deleteRow: "Do you really want to delete this row?"
```

**Comment by czosel** [comment]
Just nitpicking about the wording, not sure if you agree with it?

**Comment by fkm** [comment]
Sounds good :+1:

**Comment by anehx** [comment]
@fkm please rebase, #518 resolved the build issue :smile:

**Pull request merged**